frontend/views/containers/contributions/ContributionsOverviewWidget.vue

```diff
@@ -3,10 +3,10 @@ page-section(v-if='showHistory' :title='L("Support History")')
   i18n.support-history.button.is-outlined.is-small(
     tag='span'
     @click='toggleHistory'
-  ) This month's overview
+  ) This period's overview
   support-history
 
-page-section(v-else :title='L("This months overview")')
+page-section(v-else :title='L("This period overview")')
```

Comment:
Hmm, maybe we should keep these as "This month's overview"? (If we revert the change, be sure to revert it elsewhere.

Only thing is - my leaving it as "period" instead of month, we leave open the possibility for shorter or longer periods in the future if people want that.